src/ext.js

```diff
@@ -153,6 +153,33 @@ export default function ext({ translator }) {
                   defaultValue: false,
                   show: (data) => checkShowAction(data, 'partial'),
                 },
+                // adds automation to actions and adds a dropdown property panel
+                // item to select the automation for the button to trigger
+                automation: {
+                  type: 'string',
+                  component: 'dropdown',
+                  label: 'Select automation',
+                  ref: 'automation',
+                  options: async () => {
+                    const automations = await fetch('../api/v1/items?resourceType=automation')
+                      .then(response => response.json());
+                    return automations.data.map((blend) => ({
+                      value: blend.id,
+                      label: blend.name
+                    }));
+                  },
+                  show: (data) => checkShowAction(data, 'automation'),
+                },
+                // Boolean property to instruct the automation action to create a
+                // bookmark and send it to the selected automation in the
+                // property panel.
+                automationPostData: {
+                  type: 'boolean',
+                  ref: 'automationPostData',
+                  label: 'Send current selections to automation?',
```

Comment:
This would need to be translated. automationPostData and Send current selections to automation does not really make sense together.

Reply:
I think I don't know what I'm doing here because I don't understand what you mean with your comment.